BaseFilter/sections/general_url.txt

```diff
@@ -19,6 +19,10 @@
 ! /asset/jquery/slim-3.2.min.js$domain=avple.video|javhdfree.icu|nekolink.site|mambast.tk|netflav.com|fembed-hd.com|ndrama.xyz|pornhole.club|ffem.club|jvembed.com|dzmdplay.xyz|jav247.top|yuistream.xyz|javfu.net|iframe2videos.xyz|viplayer.cc|watchjavnow.xyz|cdn-myhdjav.info|embed-media.com|fembed9hd.com|cloudrls.com|vidgo.top
 !
 !
+! AS CNAME ex. https://www.47206262.xyz/script/jmty.jp.js
+/^https:\/\/www\.\d{8}\.xyz\/script\/[0-9A-Za-z.-]+\.[A-Za-z-]{2,}\.js$/$script,third-party
+!+ PLATFORM(ios, ext_safari)
+/^https:\/\/www\.[0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9]\.xyz\/script\/[0-9A-Za-z.-]+\.[A-Za-z-][A-Za-z-]+\.js$/$script,third-party
```

Comment:
Ok, checked in Safari. I think they can be moved to `! START: Ad-Shield ad reinsertion` section, like generic elemhides.

Reply:
Moved